FrontEnd/src/components/AboutPage/MeetTheTeam/MemberInfoCard/MemberInfoCard.module.scss

```diff
@@ -0,0 +1,77 @@
+.card {
+  width: 22.125rem;
+  height: 11.125rem;
+  border-radius: 0.5rem;
+  overflow: hidden;
+  border: 0.25rem solid #f8e6d8 !important;
```

Comment:
Check if #f8e6d8 is already a variable (you can check the Frontend/src/assets/scss/utils/global_variables.scss file). If not, consider: will this be used several times? Should it be a variable? If that color is only used in this file, then you can set the variable locally (in this file, like this: $var-name: value)